src/keydra/clients/splunk.py

```diff
@@ -205,6 +210,37 @@ def change_passwd(self, username, oldpasswd, newpasswd):
 
         return True
 
+    def rotate_hectoken(self, inputname):
+        '''
+        Rotate a Splunk HEC token
+
+        :param inputname: The name of the HTTP input
+        :type inputname: :class:`string`
+
+        :returns: New token value if successful
+        :rtype: :class:`string`
+        '''
+        response = self._service.get(
+            '/services/data/inputs/http',
+            output_mode='json'
+        )
+        inputs = json.loads(response['body'].read())
+
+        for entry in inputs['entry']:
+            if entry['name'] == 'http://'+inputname:
```

Comment:
Based on the example in `docsrc/providers.rst`, will `entry['name']` be something like `http://hec1`? 🤔